src/test/java/org/springframework/data/elasticsearch/entities/Chat.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.elasticsearch.entities;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.Documents;
+
+/**
+ * @author zzt
+ */
+@Setter
+@Getter
+@NoArgsConstructor
+@Builder
+@Documents(indexPattern = "chat_#{indexNameVar.toString()}", type = "chat")
```

Comment:
Can this reference a field in the document that contains date information?It feels like the reference should be to something in the object to be persisted

Reply:
@Field(name="createDate" store=false index= false)
private Date createDate

or @IndexVar 
private Date createDate